routes/articles.js

```diff
@@ -89,22 +90,4 @@ router.post('/article/:id/liked', (req, res) => {
     })
 })
 
-router.post('articles/:id/publish', (req, res) => {
-    const db = req.app.get('db')
-    const { id } = req.params
-    db.query('UPDATE articles SET isPublished = 1 WHERE id = ?', [id], (error => {
-        if(error) throw error
-        res.json()
-    }))
-})
-
-router.post('articles/:id/unpublish', (req, res) => {
-    const db = req.app.get('db')
-    const { id } = req.params
-    db.query('UPDATE articles SET isPublished = 0 WHERE id = ?', [id], (error => {
-        if(error) throw error
-        res.json()
-    }))
-})
-
```

Comment:
Why is this no longer needed?

Reply:
We replaced the `/publish` and `/unpublish` calls with just simply using the edit article endpoint (`POST /articles/article/:id`).